internal/storage/v2/grpc/trace_storage.proto

```diff
@@ -129,6 +129,14 @@ message FindTraceIDsResponse {
   repeated FoundTraceID trace_ids = 1;
 }
 
+// Chunking requirements:
+// - A single chunk MUST NOT contain spans from multiple traces.
```

Comment:
the use of the word "chunk" is misleading here. Since the type is called xxChunk one might interpret that the whole chunk (multiple `v1.TracesData` entries) must be for the same trace, which is not the case.